### PR TITLE
issue-109 Rank API 

### PR DESCRIPTION
**Pull request by awsl-dbq**
1. translate Rank API
2. fix the problems reviewed in Task API

### CODE REVIEW

**Comment by imbajin** [comment]
Thanks, seems u need rebase the `website` branch & solve conflicts first?

File: content/en/docs/clients/restful-api/rank.md [hunk 1]
```diff
@@ -4,28 +4,35 @@ linkTitle: "Rank"
...
 
 #### 4.2.1 Personal Rank API
 
-Personal Rank 算法典型场景是用于推荐应用中, 根据某个点现有的出边, 推荐具有相近 / 相同关系的其他点,
-比如根据某个人的阅读记录 / 习惯, 向它推荐其他可能感兴趣的书, 或潜在的书友, 举例如下:
-1. 假设给定 1个 Person 点 是 tom, 它喜欢 `a,b,c,d,e` 5本书, 我们的想给 tom 推荐一些书友, 以及一些书, 最容易的想法就是看看还有哪些人喜欢过这些书 (共同兴趣)
-2. 那么此时, 需要有其它的 Person 点比如 neo, 他喜欢 `b,d,f` 3本书, 以及 jay, 它喜欢 `c,d,e,g` 4本书, lee 它喜欢 `a,d,e,f` 4本书
-3. 由于 tom 已经看过的书不需要重复推荐, 所以返回结果里应该期望推荐有共同喜好的其他书友看过, 但 tom 没看过的书, 比如推荐 "f"  和 "g" 书, 且优先级 f > g
-4. 此时再计算 tom 的个性化 rank 值, 就会返回排序后 TopN 推荐的 书友 + 书 的结果了 (如果只需要推荐的书, 选择 OTHER_LABEL 即可)
+A typical scenario for `Personal Rank` algorithm is in recommendation application. According to the out edges of some vertex,
```
Comment (javeme) [inline content/en/docs/clients/restful-api/rank.md:17]: According to the out edges of a vertex, recommend some other vertices that having the same or similar edges.

File: content/en/docs/clients/restful-api/rank.md [hunk 2]
```diff
@@ -4,28 +4,35 @@ linkTitle: "Rank"
...
+recommend some other vertex having the same or similar  edges.
+
+Here is a use case.
```
Comment (javeme) [inline content/en/docs/clients/restful-api/rank.md:20]: "Here is a use case:"

File: content/en/docs/clients/restful-api/rank.md [hunk 3]
```diff
@@ -4,28 +4,35 @@ linkTitle: "Rank"
...
+According to someone's reading habiit or reading history, we can recommend some books he may interested or some book pal.
+
+For Example:
+1. Suppose we have a vertex,Person type, and named tom.He like 5 books `a,b,c,d,e` .If we want to recommend some book pal and books for tom, a most easier idea is let's check whoever also liked these books (common hobby based).
+2. Now, we need someone else,like neo, he like three books `b,d,f`. And Jay, he like 4 books `c,d,e,g`, and Lee, he also like 4 books `a,d,e,f`.
+3. For we don't need to recommend books tom already readed, the recommend-list should only contain the books tom's book pal already readed but tom haven't read yet. Such as book "f" and "g", and with priority f > g.
+4. Now, we recompute tom's personal rank value, we will get a sorted TopN book pal or book recommend-list. (Choose OTHER_LABEL,for Only Book purpose)
```
Comment (javeme) [inline content/en/docs/clients/restful-api/rank.md:27]: check "," following a space

**Review by javeme — COMMENTED** [review]
Thanks for your contribution. some comments

File: content/en/docs/clients/restful-api/task.md [hunk 4]
```diff
@@ -91,7 +91,7 @@ DELETE http://localhost:8080/graphs/hugegraph/tasks/2
 204
 ```
 
-#### 7.1.4 取消某个异步任务，**该异步任务必须具有处理中断的能力**
+#### 7.1.4 Cancel an async task**the task should be able to be canceled**
```
Comment (javeme) [inline content/en/docs/clients/restful-api/task.md:94]: missing comma "Cancel an async task"

**Comment by imbajin** [comment]
Hi, could u submit a new PR to `master` branch 

even if u don't have time to address comments?

**Comment by imbajin** [comment]
due to the `website` branch has deleted, I'll address the pr into the master soon

**Pull request closed without merging**